client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/hooks/useDraftPrescriptionLines.tsx

```diff
@@ -22,6 +23,13 @@ export interface UseDraftPrescriptionLinesControl
   extends UseDraftStockOutLinesControl {
   updateNotes: (note: string) => void;
 }
+export const createDraftPrescriptionLine = ({
```

Comment:
could be helpful to add a comment explaining why we don't want to use `createDraftStockOutLine` here, which is, I assume, because we don't want the `availableNumberOfPacks` messed around with